File: New Restaurantelar API/src/main/java/com/salesianos/triana/proyecto/Restaurantelar/dto/CreateValorationDto.java
package com.salesianos.triana.proyecto.Restaurantelar.dto;

import com.salesianos.triana.proyecto.Restaurantelar.model.Plato;
import com.salesianos.triana.proyecto.Restaurantelar.security.user.UserEntity;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class CreateValorationDto {

    private String criticEmail;
    private Long platoId;
    private double rating;
    private String comments;

}

File: new_restaurantelar_admin/src/app/models/create_valoration_dto.ts
export class CreatePropertyDto {
    criticEmail: string;
    platoId: number;
    rating: number;
    comments: string;
    
    constructor() {
      this.criticEmail = '';
      this.platoId = 0;
      this.rating = 0;
      this.comments = '';
      }
  }